src/components/text-editor/link-menu/editor-link-menu.tsx

```diff
@@ -113,7 +113,7 @@ export class TextEditorLinkMenu {
             <limel-input-field
                 label={this.getTranslation('editor-link-menu.link')}
                 value={this.link?.href || ''}
-                type="url"
```

Comment:
I don't know what problems you experienced, but this could have some accessibility benefits, as well as mobile-friendly on-screen keyboards. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url 

Maybe we should allow custom link-validations possibilities for the component, instead of removing it and changing the type to `text`.

Reply:
Also, i noticed that you cannot press <kbd>Enter</kbd> when the focus is on this "open link" button. Not sure if it's related, or of it's because of the changes introduced here: [`2da6dba` (#3023)](https://github.com/Lundalogik/lime-elements/pull/3023/commits/2da6dbac7aa4ec89a7e14d5304ce83743ecf30fd)
<img width="447" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/579fad6f-0a16-461a-8080-abd8a39ad31e">

Reply:
> I don't know what problems you experienced, but this...

It's unfortunate that the input field serves double duties as a relative and absolute url validator. For relative urls, native url validation will tell us that the input field is invalid.

> Maybe we should allow custom link-validations possibilities for the component, instead...

I don't fully understand. Do you mean native client side form validation as we have now?

> Also, i noticed that...

I'll look into the open link button, it might be a small fix.

Reply:
Or we could use an aria label 🤷🏽

Reply:
> Also, i noticed that you cannot press Enter when the focus is on this "open link" button. Not sure...

I couldn't reproduce this in latest main or on this branch 😅

Reply:
https://github.com/Lundalogik/lime-elements/assets/5563232/978ad15c-ecf7-462d-b3b4-abe6cc65bc64

Reply:
> > Maybe we should allow custom link-validations possibilities for the component, instead of removing it and changing the type to text.
> 
> There's already custom link validation in place.

But I'm unable to produce an invalid url. 😅 
And when I'm looking at the implementation of the method `validateLink` it only validates that it's a non-empty string really.

Still, I think it's quite difficult to produce an invalid link when accepting relative uri:s.